pkg/types/dsse/dsse.go

```diff
@@ -61,7 +62,7 @@ func ToLogEntry(ds *pb.DSSERequestV0_0_2) (*pb.DSSELogEntryV0_0_2, error) {
 			allPubKeyBytes[v] = cert.RawBytes
 		}
 	}
-	signerVerifiers, err := verifyEnvelope(allPubKeyBytes, ds.Envelope)
+	signerVerifiers, err := verifyEnvelope(allPubKeyBytes, ds.Envelope, crypto.SHA256, algorithmRegistry)
```

Comment:
Should merge https://github.com/sigstore/rekor-tiles/pull/179 first to avoid hardcoding the algorithm

Reply:
we're reworking PR: #179 as described in issue: #176 , so lets hardcode for now.